src/mono/wasi/testassets/LibraryMode.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Runtime.InteropServices;
+public unsafe class Test
+{
+    [UnmanagedCallersOnly(EntryPoint = "MyCallback")]
```

Comment:
It's same as browser's `LibraryMode.cs` but I can't see a good place for common test assets location. Each: wasi and browser has their own `BuildEnvironment.TestAssetsPath`. Maybe after refactoring wasi WBT we could solve such duplicates?